### PR TITLE
New Buttons

### LINKED ISSUE #337: Buttons needs to change to match specs
Main specs
https://www.figma.com/file/37rEn9Up8BK377t7KFQZ91/Bancor-Design-System?node-id=103%3A1425

Buttons to align (Probably others after changing)

Deposit button
https://www.figma.com/file/fwADI9wqDrRAdlMX8EddCw/Bancor-v3?node-id=12678%3A385376

Wallet button
https://www.figma.com/file/fwADI9wqDrRAdlMX8EddCw/?node-id=10803%3A320511

### PR DESCRIPTION
**Pull request by RanCohenn**
Fixes #337 #351 #302 #322

### CODE REVIEW

**Comment by zavelevsky** [comment]
V2 deposit pool selection is broken

**Comment by ashachaf** [comment]
image

**Comment by ashachaf** [comment]
image

**Comment by ashachaf** [comment]
image

**Comment by ashachaf** [comment]
image

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -1,55 +1,78 @@
...
+    case ButtonVariant.Primary:
+      return `bg-primary hover:bg-primary-dark disabled:bg-fog dark:disabled:bg-grey text-white ${disabledText}`;
+    case ButtonVariant.Secondary:
+      return `bg-black hover:bg-grey disabled:bg-fog  dark:bg-white dark:hover:bg-silver dark:disabled:bg-grey text-white dark:text-charcoal ${disabledText}`;
+    case ButtonVariant.Tertiary:
+      return 'bg-fog hover:bg-black disabled:bg-fog text-charcoal hover:text-white dark:text-white dark:hover:text-charcoal disabled:text-black-low disabled:text-black-low dark:disabled:text-white-low';
+    case ButtonVariant.Error:
+      return '';
+    case ButtonVariant.Warning:
+      return '';
```
Comment (pingustar) [inline src/components/button/Button.tsx:40]: warning and error buttons are missing

**Pull request merged**